### ISSUE TITLE
don't recomment raw Unicode strings; false positive about \U escape

### ISSUE THREAD

**Issue by birkenfeld** [thatch/regexlint#23]
```
E112:ResourceLexer:string:L42: Regex parser does not handle unicode, use ur"" string.
              (r'(\\x[0-9a-fA-F]{2}|\\u[0-9a-fA-F]{4}|\\U00[0-9a-fA-F]{6}|'
                                                       ^ here
```

This is doubly wrong:
- the regex contains no Unicode escape, but matches a literal `\U...`
- ur"" strings are not allowed by Python 3, we can't use them.
